front/src/components/mainPage/SlideBar.jsx

```diff
@@ -1,15 +1,12 @@
 import React, { useEffect, useRef, useState } from 'react';
 import styled from 'styled-components';
 import Slide from './Slide';
-import { ButtonUI } from '../public/StyledComponent';
-import { useNavigate } from 'react-router-dom';
 
 const TOTAL_SLIDES = 2; // 전체 슬라이드 개수(총3개. 배열로 계산)
 
 export default function SlideBar() {
   const [currentSlide, setCurrentSlide] = useState(0);
   const slideRef = useRef(null);
-  const navigate = useNavigate();
 
   // Next 버튼 클릭 시
   const NextSlide = () => {
```

Comment:
```suggestion
  const handleSlideNext= () => {
```
`camelCase` 컨벤션 지켜주시고 핸들러 이름도 이정도가 더 나을거같네요

Reply:
확인했습니다